File: year2020/month12/day1214/ms_1214.cs
using System;

namespace algorithm_121
{
    class ms_1214
    {
        static string solution(string s) 
        {
            string answer = "";
            int index = 0;
            
            for(int i = 0; i < s.Length; i++)
            {
                char temp = s[i];

                if(temp == ' ')
                {
                    index = 0;
                    answer += temp;
                    continue;
                }

                if(index%2 == 0)
                {
                    answer += Char.ToUpper(temp);
                }
                else
                {
                    answer += Char.ToLower(temp);
                }

                index++;
            }

            return answer;
        }

        static void Main(string[] args)
        {
            String s = Console.ReadLine();

            if(1 <= s.Length)
            {
                Console.WriteLine(solution(s));
            }
        }
    }
}

File: year2020/month12/day1214/ms_1214.cpp
#include <iostream>
#include <string>

using namespace std;

string solution(string s);

int main() {
	string s;

	getline(cin, s);

	if (1 <= s.length()) {
		cout << solution(s) << endl;
	}

	return 0;
}

string solution(string s) {
	string answer = "";
	int index = 0;

	for (int i = 0; i < s.length(); i++) {
		char temp = s[i];

		if (temp == ' ') {
			index = 0;
			answer += temp;
			continue;
		}

		if (index % 2 == 0) {
			answer += toupper(temp);
		}
		else {
			answer += tolower(temp);
		}

		index++;
	}

	return answer;
}